pkg/scheduler/framework/plugins/noderesources/fit.go

```diff
@@ -421,8 +415,18 @@ func (f *Fit) isSchedulableAfterNodeChange(logger klog.Logger, pod *v1.Pod, oldO
 	if err != nil {
 		return fwk.Queue, err
 	}
+	// Use the DRA manager's extended resource cache for event handlers
+	var draManager fwk.SharedDRAManager
+	if f.enableDRAExtendedResource {
+		draManager = f.handle.SharedDRAManager()
+	}
```

Comment:
Why the feature gate check? Shouldn't `f.handle.SharedDRAManager` return nil when the feature gate is off?

Reply:
Feature-gate check can be removed here